### ISSUE TITLE
Merging calls

### ISSUE THREAD

**Issue by nandobonf** [tf2/CNest#11]
Hi, I read with great interest the paper on Cell Genomics and I was wondering if you can also provide a script to generate the merged copy number events (CNVEs) as described in the paper for better reproducibility.
Thanks!

**Comment by nandobonf** [comment]
@tf2 any update on this? Thank you!

**Comment by tf2** [comment]
yep - many apologies for the slow reply - on it... you will notice there is a R package push just now in the "src" dir called "recipeB" -- please try and install that as a first step... there is a function called "rbreak" which is very flexible and can do the merging you require. I will provide example of how it works when you let me know its installed ok (it requires a few parameters to be set and a speific input format)....

**Comment by nandobonf** [comment]
thanks and apologies for my late reply (I missed somehow the notification). However, I'm still not able to make it work, i get many warnings in R. I will post you later the logs. Thanks!
